wlct/templates/pr.html

```diff
@@ -39,6 +39,29 @@ <h4>{{ tournament.name }} &nbsp;
                 {% endif %}
 
     {% if tournament.created_by.token == request.session.token and not tournament.has_started %}
```

Comment:
Does this mean you cannot copy a season that has started?

Reply:
Good point, moved into new if block to check for created_by token only.